### ISSUE TITLE
`requireHyphenBeforeParamDescription` fixer insert hyphen to wrong place

### ISSUE THREAD

**Issue by golopot** [gajus/eslint-plugin-jsdoc#176]
This is reported correctly:
```javascript
/**
 * @param a - foo
 * @param b foo misses hyphen
 */
```

But fixer fixes a wrong place.
```javascript
// fixed code
/** 
 * @param a - - foo
 * @param b foo
 */
```

This happens because the fixer is written like this: `str.replace('foo', '- foo')`

Ideally the fixer should find the location of the tag description, and just insert hyphen there. Unfortunately `comment-parser` does not provide the location of the description.
